src/components/language-selector.js

```diff
@@ -1,51 +1,82 @@
 import React, { useState } from "react";
-import { Grid, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { navigate } from "gatsby";
+import { Button, Grid, Menu, NavDropDownButton } from "@trussworks/react-uswds";
 import { useTranslation } from "react-i18next";
 import FeatherIcon from "feather-icons-react";
-import { Link } from ".";
-import languages from "../constants/languages";
+import { languages } from "../constants/languages";
 
 function LanguageSelector({ slug }) {
   const { i18n } = useTranslation();
 
   const [isOpen, setIsOpen] = useState(false);
 
+  const handleClick = (langKey) => {
+    i18n.changeLanguage(langKey);
+    navigate(`/${langKey}/${slug}`);
+  };
+
+  const languageButtonGroup = languages.map((language) => (
```

Comment:
What is the difference between this and what's on line 39?

Reply:
Where are you suggesting a reference the index? I was thinking I'd reference the array length to conditionally set the class name

I'll work on combining languageButtonGroup and LanguageMenuItems!

Reply:
In line 31, you currently have it as `const languageMenuItems = languages.map((language) => (` when you can have it as `const languageMenuItems = languages.map((language, index) => (` to keep track of the current index. This way, instead of repeating your code on lines 39-48, you can have it simply as `{languageButtonGroup[i]}` or something similar if not exact.

Reply:
Oh I see what you're saying. I don't think I need to keep track of the index. `languageButtonGroup` and `languageMenuItems` both are the return values of looping through `languages`. The only difference is the class name on the container `div`. I can conditionally set the class name I think. And replace `languageButtonGroup`  with `languageMenuItems` on line 79

Reply:
I combined the two by conditionally setting the class names in the `styling-updates` PR like this:

```
const languageMenuItems = languages.map((language) => (
    <div
      className={
        languages.length <= 5
          ? "banner__language-selector-item"
          : language.isRtoL
          ? "banner__language-selector-item--RtoL"
          : "banner__language-selector-item--LtoR"
      }
    >
etc...
```

Reply:
Nice! And you're getting rid of `languageButtonGroup`? Also, is it necessary to add the class `"banner__language-selector-item--LtoR"`? Can you get away with having an empty string?

Reply:
Yup! `languageButtonGroup` was the same thing as `languageMenuItems` except for the different class name, so I combined them :) 

And I do use specific styling for `banner__language-selector-item--LtoR`